packages/flutter/lib/src/material/tabs.dart

```diff
@@ -821,7 +829,9 @@ class _TabBarState extends State<TabBar> {
     //
     // The material's color might be null (if it's a transparency). In that case
     // there's no good way for us to find out what the color is so we don't.
-    if (color.value == Material.of(context)?.color?.value)
+    // TODO(xu-baolin): Remove this code without breaking something.
```

Comment:
I add a `TODO` here for the possible long-term solutions in the future

Reply:
I think adding a `TODO` here is good, let's just make it a little more descriptive since we might never get back to it, but a future Flutter developer might. I'm thinking:
```dart
    // TODO(xu-baolin): Remove automatic adjustment to white color indicator with a better long-term solution.
```